### PR TITLE
Remove using files with .scsi extension

### PR DESCRIPTION
**Pull request by KovalM**

### CODE REVIEW

**Review by MikhailSadovsky — APPROVED** [review]

**Review by deniskoronchik — CHANGES_REQUESTED** [review]
How that changes were made?
How that changes were tested?

**Comment by deniskoronchik** [comment]
This PR contains 3 different PR's:
- Remove *.scsi
- Add action classes
- Fix system identifiers

**Comment by KovalM** [comment]
> This PR contains 3 different PR's:
> 
> * Remove *.scsi
> * Add action classes
> * Fix system identifiers

Divided on 3 PR's: #91 #92 #93

File: ui/languages/ui_external_languages.scs [hunk 1]
```diff
@@ -1,3 +1,9 @@
 ui_external_languages <- sc_node_not_relation;;
 
 ui_external_languages -> scg_code; scs_code;;
+
+nrel_answer
```
Comment (ShunkevichDV) [inline ui/languages/ui_external_languages.scs:5]: Why is it here?

**Review by ShunkevichDV — APPROVED** [review]
May be we should create issue to describe nrel_asnwer and nrel_authors in other files

File: ui/languages/ui_external_languages.scs [hunk 2]
```diff
@@ -1,3 +1,9 @@
...
+<- sc_node_norole_relation;;
+
+nrel_authors
+<- sc_node_norole_relation;;
```
Comment (deniskoronchik) [inline ui/languages/ui_external_languages.scs:9]: Same question

File: ui/ui_nrel_command_template.scs [hunk 3]
```diff
@@ -1,2 +1,10 @@
...
+ui_nrel_command_template
 
+=> nrel_main_idtf:
+    [обобщенная формулировка команды пользовательского интерфейса*]
+    (* <- lang_ru;; *);
+    [template for user interface command*]
+    (* <- lang_en;; *);
+
+<- quasybinary_relation;
+<- sc_node_norole_relation;;
```
Comment (deniskoronchik) [inline ui/ui_nrel_command_template.scs:10]:
```
ui_nrel_command_template
 <- sc_node_norole_relation;;
.....
```

File: ui/languages/ui_external_languages.scs (hunk 1 above)
Comment (KovalM) [reply ui/languages/ui_external_languages.scs]: Merge sources mistake. Removed.
I created separate issue for describing nrel_answer relation: https://github.com/ostis-dev/ims.ostis.kb/issues/94

File: ui/languages/ui_external_languages.scs (hunk 2 above)
Comment (KovalM) [reply ui/languages/ui_external_languages.scs]: Merge sources mistake. Removed.

**Comment by KovalM** [comment]
> How that changes were made?
> How that changes were tested?

1. Changed was made by python script. I can provide it, if you need.
2. We use ostis-web-platform version 0.5.0 for test knowledge base in local version of ims.ostis.net. Replace ims.ostis.kb_copy to ims.ostis.kb in repo.path file for testing full KB.

**Comment by KovalM** [comment]
Discussed. Changes will be radically revised

**Pull request closed without merging**